blocks/marquee/marquee.css

```diff
@@ -324,6 +323,14 @@ main .section-wrapper .marquee-wrapper {
 }
 
 @media screen and (min-width: 600px) {
+    main .section-wrapper .marquee-wrapper {
+        max-width: none;
+        padding: 0;
```

Comment:
I think this is the only line of code you need to change. It should be `padding: 0 2rem` to match `main .section-wrapper > div`. 
You'll see in the screenshot the marquee css is overwriting `main .section-wrapper > div` css, which zero's out the padding. 

You _should_ be able to remove all other css changes other than this. 

![image](https://user-images.githubusercontent.com/2539954/223596951-288f6005-8c77-4b97-b97d-0c0bf95fcf3e.png)

Reply:
Following a huddle conversation between me and @rgclayton, we have come to the conclusion that the other parts of this pull request are also necessary, therefore this conversation is resolved!